### PR TITLE
Adds Schlock Mercenary

### LINKED ISSUE #1999: [Source Request] Schlock Mercenary
https://www.schlockmercenary.com/

Thank you.

### PR DESCRIPTION
**Pull request by dakriy**
Closes #1999

New extension for [Schlock Mercenary](https://www.schlockmercenary.com)

### CODE REVIEW

File: src/en/schlockmercenary/src/eu/kanade/tachiyomi/extension/en/schlockmercenary/Schlockmercenary.kt [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    override fun popularMangaFromElement(element: Element): SManga {
+        val book = element.select("h4 > a").first()
+        return SManga.create().apply {
+            url = book.attr("href")
+            title = book.text()
+            artist = "Howard Tayler"
+            author = "Howard Tayler"
+            // I'm not sure how to determine this quite yet, so I'll just hardcode the latest one to get something done
+//            status = if (book.text().contains("Book 20")) SManga.ONGOING else SManga.COMPLETED
```
Comment (CarlosEsco) [inline src/en/schlockmercenary/src/eu/kanade/tachiyomi/extension/en/schlockmercenary/Schlockmercenary.kt:56]: remove this comment

Comment (dakriy) [reply src/en/schlockmercenary/src/eu/kanade/tachiyomi/extension/en/schlockmercenary/Schlockmercenary.kt:56]: It's a WIP. It's not completed yet. But, noted. Thanks.

Comment (CarlosEsco) [reply src/en/schlockmercenary/src/eu/kanade/tachiyomi/extension/en/schlockmercenary/Schlockmercenary.kt:56]: Somehow missed the Wip 😂

**Comment by dakriy** [comment]
@CarlosEsco Would you mind taking another look at this?

**Pull request merged**